Patches/WWII Soviet Faction/WWIISovietFaction_CE_Patch_Clothing.xml

```diff
@@ -0,0 +1,154 @@
+<Patch>
+	<Operation Class="PatchOperationSequence">
+		<success>Always</success>
+		<operations>
+
+			<li Class="CombatExtended.PatchOperationFindMod">
+				<modName>WWII Soviet Faction (1.0)</modName>
+			</li>
+
+			<!-- ========== M73 Uniform ========== -->
+
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="Apparel_M73_Uniform"]/statBases</xpath>
+				<value>
+					<Bulk>8</Bulk>
+					<WornBulk>4</WornBulk>
+				</value>
+			</li>
+
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="Apparel_M73_Uniform"]/statBases/ArmorRating_Sharp</xpath>
+				<value>
+					<ArmorRating_Sharp>0.3</ArmorRating_Sharp>
+				</value>
+			</li>
+
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="Apparel_M73_Uniform"]/statBases/ArmorRating_Blunt</xpath>
+				<value>
+					<ArmorRating_Blunt>0.075</ArmorRating_Blunt>
+				</value>
+			</li>	
+
+			<!-- ========== Harness ========== -->
+
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="Apparel_M73_Harness"]/statBases</xpath>
+				<value>
+					<Bulk>3</Bulk>
+					<WornBulk>3</WornBulk>
+				</value>
+			</li>
+
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="Apparel_M73_Harness"]/equippedStatOffsets</xpath>
+				<value>
+					<CarryBulk>18</CarryBulk>
+				</value>
+			</li>
+
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="Apparel_M73_Harness"]/apparel</xpath>
+				<value>
+					<apparel>
+						<bodyPartGroups>
+							<li>Shoulders</li>
+						</bodyPartGroups>
+						<layers>
+							<li>Webbing</li>
+						</layers>
+						<tags>
+							<li>WWII_Soviet_Harness</li>
+						</tags>
+						<wornGraphicPath>Things/Apparel/Belts/M73Harness</wornGraphicPath>
+					</apparel>
+				</value>
+			</li>
+
+			<!-- ========== M35 Uniform ========== -->
+
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="Apparel_M35_Uniform"]/statBases</xpath>
+				<value>
+					<Bulk>8</Bulk>
+					<WornBulk>4</WornBulk>
+				</value>
+			</li>
+
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="Apparel_M35_Uniform"]/statBases/ArmorRating_Sharp</xpath>
+				<value>
+					<ArmorRating_Sharp>0.3</ArmorRating_Sharp>
+				</value>
+			</li>
+
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="Apparel_M35_Uniform"]/statBases/ArmorRating_Blunt</xpath>
+				<value>
+					<ArmorRating_Blunt>0.075</ArmorRating_Blunt>
+				</value>
+			</li>
+
+			<!-- ========== Vest ========== -->			
+
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="Apparel_SN-42Armor"]/statBases</xpath>
+				<value>
+					<StuffEffectMultiplierArmor>2</StuffEffectMultiplierArmor>
+					<Bulk>8</Bulk>
+					<WornBulk>5.5</WornBulk>
+				</value>
+			</li>
+
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="Apparel_SN-42Armor"]/statBases/ArmorRating_Sharp</xpath>
+				<value>
+					<ArmorRating_Sharp>10</ArmorRating_Sharp>
```

Comment:
This is the equivalent thickness of the armor in mm of RHA (essentially, armor-grade steel). Since we already know the SN-42 is just made of 2mm thick steel, set:

```
<ArmorRating_Sharp>2</ArmorRating_Sharp>
 <ArmorRating_Blunt>3</ArmorRating_Blunt>
```
(We know the blunt armor since for Steel: `<StuffPower_Armor_Blunt>1.5</StuffPower_Armor_Blunt>`

Reply:
Fixed in db3fbab